apps/docs/src/app/docs/running-blobscan-locally/page.md

```diff
@@ -59,10 +59,16 @@ Then run the development command:
 pnpm dev
 ```
 
-Lastly, create the database schema:
+Now apply all database migrations:
 
 ```shell
-pnpm db:generate
+pnpm db:migrate
+```
+
+Lastly, you can add some fixture data:
+
+```shell
+pnpm db:seed
```

Comment:
These lines need to be thrown in `/packages/db/` in order to work. We could make these commands accessible on the root path or add these information in the doc

Reply:
It's a dev-related command. I'd navigate to the `db` package to execute it